doc/decisions/operation_sequences.md

```diff
@@ -0,0 +1,92 @@
+# KDB operation sequences
```

Comment:
Should be same as file. I think `Illegal Sequences` are a better name.

Reply:
It's not. This decision is not pureley about illegal sequences, but as @kodebach mentiones also about the semantical meaning of allowed sequences.

Reply:
I am not sure if this is a good idea. If there is also unclear semantics, this should be in separate decisions. Are these semantics also of relevance for #4554? It is important that we find out the connections between the decisions, so that we solve them in the right order.

Btw. you are doing a great job here. This is really an operation in the heart of Elektra. :heart_decoration:

Reply:
See my recent response in the root thread of this PR. The "semantics" don't refer to the semantics of `kdbGet`/`kdbSet` on their own, but purely to the semantics of sequences, i.e. how they interact when used on the same handle.